src/main/java/core/basesyntax/controller/ConsoleHandler.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.interfaces.BetDao;
+import core.basesyntax.dao.interfaces.UserDao;
+import core.basesyntax.lib.Inject;
+import core.basesyntax.model.Bet;
+import core.basesyntax.model.User;
+import java.util.Scanner;
+
+public class ConsoleHandler {
+
+    @Inject
+    private BetDao betDao;
+    @Inject
+    private UserDao userDao;
+
+    public void handle() {
+        Scanner scanner = new Scanner(System.in);
+
+        while (true) {
+            System.out.println("Please, enter login and password for your user!");
+            String inputUser = scanner.nextLine();
+            System.out.println("Please, enter value and risk for your bet!");
+            String inputBet = scanner.nextLine();
+            if (inputUser.equalsIgnoreCase("q") || inputBet.equalsIgnoreCase("q")) {
+                break;
```

Comment:
I think we can put return instead of break; it will be more correct

Reply:
what if we have add some actions after loop